spec/spec_helper.rb

```diff
@@ -3,3 +3,17 @@
 root_dir = File.expand_path('..', __dir__)
 IMAGES_DIR = File.join(root_dir, 'doc/ex/images')
 SUPPORT_DIR = File.join(root_dir, 'spec', 'support')
+
+module Magick
```

Comment:
Is there a strong case for using a `module` here? I might just keep it top-level like the `unsupported?` method below.